### PR TITLE
26 architecture overview

### PR DESCRIPTION
**Pull request by xdoo**
see #26

### CODE REVIEW

**Comment by xdoo** [comment]
@dominikhorn93 ready for review

File: content/resources/documentation/architecture/overview.md [hunk 1]
```diff
@@ -1,74 +1,87 @@
...
-## First header
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+You can use any technology you want. There're only two preconditions:
+
+- your frontend technology must have a GraphQL client
+- your backen (services, integration artifacts) must be able to communicate with one of [Spring Cloud Streams](https://spring.io/projects/spring-cloud-stream) supported Binder implementations.
 
 <v-alert color="yellow darken-1" border="left" elevation="2" colored-border icon="mdi-robot-confused">
-This is a hint field!
+If you want use an other EventBus infrastructure than Apache Kafka, you have to configure this in your DigiWF configurations.
```
Comment (dominikhorn93) [inline content/resources/documentation/architecture/overview.md:48]: If you want to use another

File: content/resources/documentation/architecture/overview.md [hunk 2]
```diff
@@ -1,74 +1,87 @@
...
-<v-alert color="red darken-1" border="left" elevation="2" colored-border icon="mdi-robot-angry">
-This is a warn field!
-</v-alert>
+Of course - if you'll use Spring Boot in your backend components and VueJs as front end technology, you can use all cross-section components (like Spring Boot Starters, NPM components, ...) we have created for our components.
 
-<v-alert color="grey lighten-1" border="left" elevation="2" colored-border icon="mdi-robot-happy">
-This is a neutral field!
-</v-alert>
+### DigiWF Core
+DigiWF Core is the heard of DigiWF. As you can see in the section above it consists of 5 services. It has a GraphQL API to interact with ist from front end applications and a very generic API, that communicates with various backend systems over an event bus.
```
Comment (dominikhorn93) [inline content/resources/documentation/architecture/overview.md:54]: heart  ..................... to interact with its frontend apllications ....

File: content/resources/documentation/architecture/overview.md [hunk 3]
```diff
@@ -1,74 +1,87 @@
...
+If you want to integrate any kind of backend system, you can do this via the integration layer. DigiWF integration is on one side a set of predefined integration artifacts like S3, Mail, JMS or other. On the other side is DigiWF integration a toolbox, to help you to build own integration artifacts as quick as possible. 
+
+### DigiWF Tasklist
+This is a simple frontend to interact with running process instances. Every user task occurs on the tasklist and can be picked up (of course only if you have the right to). You can see the state of "your" processes and you can start new instances over the tasklist. If you don't like such a highly standardized frontend, you're free to use the components like the form builder and integrate them into your own beautiful web app. Or you can use the API directly and build all the fancy frontend stuff on your own, in the technology you like most.   
+
+### DigiWF Co-Creation
+The DigiWF Co-Creation section is the low code area. Here can a process designer draw BPMN processes, tinkering with decision tables (DMN) or create webforms per drag and drop. Deployment into different infrastructures is possible over this web app, too.
```
Comment (dominikhorn93) [inline content/resources/documentation/architecture/overview.md:63]: Here a process designer can....

**Review by dominikhorn93 — CHANGES_REQUESTED** [review]

File: content/resources/documentation/architecture/overview.md [hunk 4]
```diff
@@ -1,74 +1,87 @@
...
+<figcaption>Interaction DigWF Core & the surrounding systems</figcaption>
+</figure>
 
-``Use `code` in your Markdown file.``
+The main goal of DigiWF is to create a declarative way for process designers to interact with a technical infrastructure. For this we've created some supporting services around the opensource [camunda](https://camunda.com/) workflow:
 
----
+- Process Service: This is the service where Camunda is embedded. It is used mainly to execute the BPMN workflows.
+- Service Instance: This handles any kind of service instances. A service instance can be a process, but must not. 
+- Task Service: This service handles everything we need for human tasks (authorization, mapping, back channel, ...). The task service is backed by an Elasticsearch database, so you can do full text search over all tasks.
```
[the remark this answers is not in the document]
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:20]: Yes. Deleted.

File: content/resources/documentation/architecture/overview.md [hunk 5]
```diff
@@ -1,74 +1,87 @@
...
+- Form & Validation Service: This handles everything, that we need around forms. We store the form definition here an do all kind of form validation.
```
[the remark this answers is not in the document]
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:21]: Done.

File: content/resources/documentation/architecture/overview.md [hunk 6]
```diff
@@ -1,74 +1,87 @@
...
+artifacts. You can see the four provided DigiWF modules core, tasklist, integration and co-creation." contain 
+max-width="960" 
+src="resources/documentation/architecture/digiwf_core_modules.
+png" 
+lazy-src="preview_blue_yellow.png" ></v-img>
+<figcaption>The DigiWF Modules + possibilities to integrate custom components</figcaption>
+</figure>
 
-Bla
+The picture above shows a possible "full blown" DigiWF architecture including self crafted artifacts. Everything in blue is provided by the DigiWF project, but as you know we're open for integrations from every direction. So you could create and integrate:
```
[the remark this answers is not in the document]
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:36]: Done.

File: content/resources/documentation/architecture/overview.md [hunk 7]
```diff
@@ -1,74 +1,87 @@
...
 
-<img src="/background_oqun5d_c_scale,w_700.png" width="100%" class="light-img" alt=""/>
+- your own front ends
```
[the remark this answers is not in the document]
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:38]: Done.

File: content/resources/documentation/architecture/overview.md (hunk 1 above)
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:48]: Done.

File: content/resources/documentation/architecture/overview.md (hunk 2 above)
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:54]: Done.

File: content/resources/documentation/architecture/overview.md (hunk 3 above)
Comment (xdoo) [reply content/resources/documentation/architecture/overview.md:63]: Done.

**Comment by dominikhorn93** [comment]
Deepl + Grammarly is my way of doing architecture documentation in english 💯

**Review by dominikhorn93 — APPROVED** [review]

**Pull request merged**